org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/internal/parser/XMLParser.java

```diff
@@ -134,16 +134,17 @@ public XMLDocument parse(TextDocument document, boolean full) {
 				Node cdataNode = new Node(scanner.getTokenOffset(), text.length(), new ArrayList<>(), curr,
 						xmlDocument);// TODO: might need arraylist
 				cdataNode.isCDATA = true;
+				cdataNode.tag = "CDATA";

```

Comment:
Why do you need that? For me tag was an element, no?

Reply:
@angelozerr I figured that node.tag would be the beginning text of a tag. And I think it makes more sense for the content inside would go in to node.content.

Reply:
OK I understand your idea. Thanks for the clarification.

Reply:
@angelozerr can this be merged?

Reply:
Just one question. Do you need that only for formatting? If it that I think it should be better to fill CDATA according full flag like https://github.com/angelozerr/lsp4xml/blob/master/org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/internal/parser/XMLParser.java#L192 Indeed for instance for Content, we need just for format, so for outline (documents symbols) we don't need it, and other features like completions, etc. So to avoid having this information in memory which is used only for format, I'm using full flag. So please use full if it's only for format.

Reply:
@angelozerr You're right, I fixed CDATA content will consider if full is true.